### ISSUE TITLE
KR3 A2 bugs (v0.2)

### ISSUE THREAD

**Issue by cigumo** [cigumo/krli#50]
## Pending
- [x] High Elven Mages Timelapse has also a small chance to lead an enemy to be basically frozen leaving it untargetable but unlike the Dusttrail it cant move so it cant take a life at all. Can't be targeted by towers / SealDarklight, PitNata
- [x] bandersnatch and timelapse mod multiple glitches / AerisDraco
- [x] Faustus teleport has a small chance of making a harrasser into an unblockable and untargetable. Basically like the Dusttrail glitch but rare / SealDarklight
- [x] Twilight Harrashers can basically use their shadow jump ability while being on Dusttrails. Once they use it they cannot be blocked by barracks or targeted by either ranged heroes or towers making them very annoying to deal on Dusttrails / SealDarklight

## TBD with Ironhide
- [x] Faustus cant get harmed by Bandersnatch underneath even with the Spiked AOE althrough i dont know if it is or not but tell me. / SealDarklight
  - same as iOS.  will discuss with ironhide. (moved to cigumo/kr-love#229)

## Fixed in v0.3
- [x] game crash when clicking new heroes (not in the slot data) / AerisDraco
  - _fixed in v0.2.1_
- [x] map animation the spider abdomen near the bottom persists / AerisDraco
- [x] Reg'son's spin seems to cooldown very fast / henryhp
- [x] Heroes don't say their quotes when selected in the hero room / GeminiSparkSP
- [x] The new heroes seem to crash the game but only on the first saved slot (Im testing the second heroes on the second Saved slot) / SealDarklight
- [x] **Selling High Elven Mage will give you massive amounts of gold and it will increase which slows the game down / SealDarklight**
- [x] **Weirdwoods Clobber ability in phase 3 upgrade is the same as phase 2 description. Not sure if phase 3 deals the same or more than phase 2. it stuns eneimies from 2 seconds instead of 3 and dealing 100 instead of 125. / SealDarklight**
- [x] unspecified forest keeper crash / AerisDraco
- [x] Forest Keepers seem to freeze when they use the circle of life and Eerie gardener / SealDarklight

## Fixed in v0.9
- [x] reinforcement throwing stars also fires way too fast / henryhp
- [x] pause overlaid to victory menu / GeminiSparkSP
- [x] The sound for Denas' Mighty is off sync / GeminiSparkSP
- [x] other heroes don't get to use their spells in level 1 by wave 6, only Eridan does / GeminiSparkSP
- [x] Reg'son's Eldritch blade doesn't last long enough? its only enough for 4 hits altho i might be wrong / henryhp
- [x] Hi Hi Enha supposed to stun turrets? / Franisz
- [x] Greenlings from Bravebark turn into logs, when clicked tooltip tries to showup (disable can_click on death) / Franisz
- [x] Windows 10 resume from sleep (start/power/sleep) makes the game unresponsive to clicks / AerisDraco (moved to #51)
- [x] wrong zelot name and hp / Franisz
- [x] problems when dragging the credits page: does not go to the end / GeminiSparkSP
- [x] Hoplite's shield banging isn't loud enough / GeminiSparkSP
- [x] bandersnatches don't get affected by the 2 uppermost poison fungus in Grimmsburg / GeminiSparkSP
- [x]  All the new heroes are immune to Bandersnatches ability so i think it should be fixed. / SealDarklight
- [x] Bravebark and Xin takes no damage from Bandersnatches attacks except for spike attack / SealDarklight / SealDarklight
- [x] bandersnatch is actually slowed by the leftmost purple tomato clouds on grimmsburg map / VirtualTT
- [x] killing scourger when it starts performing its special move prevents it from spawning tower-possessing spirit, also in this case no death animation is played, scourger just disappears / VirtualTT
- [x] killing scourger with Regson's instakill attack prevents it from spawning tower-possessing spirit / VirtualTT
- [x] scourger tower-possessing spirit does not interrupt current attack of weirdwood / VirtualTT
- [x] killing fungus breeder with fire nut attack prevents it from spawning lesser fangus, also in this case no death animation is played / VirtualTT
- [x] on Crystal lake map wave 10 starts without notification / VirtualTT
- [x] crystal lake: no icon on the path to proceed to wave 10, so I have to use the button on the lower right corner / GeminiSparkSP
- [x] on Crystal lake map enemies sometimes become invisible after teleported back, i think this happens after they get frozen by little dragons
- [x] 2 uppermost poison fungi in Grimmsburg needs some tweaking, as the range of their fume animation and the actual range doesn't match at all / GeminiSparkSP
- [x] Faustus doesnt shoot his magical projectiles on the right side but he can shoot on the left side  / SealDarklight
- [x] Faustus' normal attack animation is borked. It seems as if he's going to breathe out the 3 bubbles, but then just stops. happens when enemeines come from the sides  / GeminiSparkSP
- [x] Faustus ultimate torched a non path (crystal lake) / GeminiSparkSP
  - ![screen shot 2018-08-01 at 9 36 55 am](https://user-images.githubusercontent.com/934821/43522057-79c992f0-956e-11e8-9843-4ec089763b19.jpg)
- [x] Gnol Brighters image shows Perython instead of Gnol Brighter in the Enemy encylopedia / SealDarklight
- [x] Boomshrooms will deal damage to anything regardless if its an allie or a foe. Even other boomshrooms.  / SealDarklight
- [x] Twilight Zealots have a buggy health bar / SealDarklight
- [x] Bravebarks Springseeds can heal enemies making it a really dangerous ability right now / SealDarklight
- [x] Crystal Lakes iron challenge however will make the game stuck after the enemies are slayed but if you quit it will give you a free star for some reason. / SealDarklight
- [x] Awoks cannot heal hp at all when idle / SealDarklight

## Won't fix
- [x] Golden Longbows fire rate info shows it fires fast even if the towers itself shoots pretty same speed as in the mobile. Just an info bug. / SealDarklight
  - in mobile the value reported makes no sense. the tower actually fires much faster.
- [x] Not sure if its a bug but Malicia seems to deal same damage to Denas as dealing with Regsons making her deal true damage. Intentional or not im not sure. / SealDarklight
  - yes, malicia does true damage in her area attack (melee)
- [x] Regson is Immune to Blasting Curse so no fixing for that / SealDarklight
  - regson is immune to the avenger's blasting curse right? am I missing something? 

## Suggestions
- [x] Please specify the % of golden longbow instakill chance  / GeminiSparkSP
- [x] Specify the Greenling's HP in Bravebark's skill description / GeminiSparkSP
- [x] no tooltip is shown when cursor is over tier IV tower skill upgrade button if skill is already upgraded to the maximum level
- [x] Headshot's kill animation shouldn't be giblet explosion / GeminiSparkSP
